dev/drivers/scripts/plots/aqm/jevs_aqm_grid2obs_plots.sh

```diff
@@ -52,11 +52,11 @@ export SENDDBN=NO
 export job=${PBS_JOBNAME:-jevs_${MODELNAME}_${VERIF_CASE}_${STEP}}
 export jobid=$job.${PBS_JOBID:-$$}
 
-export cycle=t${cyc}z
 #
 
 export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/${NET}/${evs_ver}
-export COMINaqm=/lfs/h2/emc/vpppg/noscrub/$USER/evs/v1.0/stats/aqm/aqm
+export EVSINaqm=/lfs/h2/emc/vpppg/noscrub/$USER/evs/v1.0/stats/aqm/aqm
```

Comment:
Let's use $NET and $evs_ver in the EVSINaqm defintion.

Reply:
Yup, missed that!!